tslint.json

```diff
@@ -66,7 +66,7 @@
     ],
     "quotemark": [true, "single", "jsx-double"],
     "radix": true,
-    "semicolon": [true, "always"],
+    "semicolon": false,
```

Comment:
This is a fairly major change that would probably break a lot of code (in terms of lint). Why do we want to change it here?